includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1919,10 +1920,23 @@ protected function check_response_for_errors( $response ) {
 					$response_code
 				);
 			} elseif ( isset( $response_body['error'] ) ) {
+				$response_body_error_code = $response_body['error']['code'] ?? null;
+				$payment_intent_status    = $response_body['error']['payment_intent']['status'] ?? null;
+
+				// Check for an 'amount_too_large' error code and a 'requires_capture' payment intent status.
+				// We redact the API error message to prevent prompting the merchant to contact Stripe support
+				// when attempting to manually capture an amount greater than what's authorized. Contacting support is unnecessary in this scenario.
+				if ( isset( $response_body_error_code ) && 'amount_too_large' === $response_body_error_code && 'requires_capture' === $payment_intent_status ) {
```

Comment:
We can omit `isset( $response_body_error_code )` - it's being covered by ` ?? ` in earlier assignment.

Reply:
Yep, you beat me to it, removed in this commit https://github.com/Automattic/woocommerce-payments/pull/7581/commits/6b36ddf58fdfce78cc5d2b78af64fa9fc88509d0